Hotify/app/src/main/java/net/serkanbal/hotify/MainActivity.java

```diff
@@ -0,0 +1,52 @@
+package net.serkanbal.hotify;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+public class MainActivity extends AppCompatActivity {
+    Button mPlay, mPause, mStop;
+    public static final String TAG = "TRIAL";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        mPlay = (Button) findViewById(R.id.play);
+        mPause = (Button) findViewById(R.id.pause);
+        mStop = (Button) findViewById(R.id.stop);
+
+        mPlay.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = new Intent(MainActivity.this, MediaPlayerService2.class);
+                intent.putExtra("action", "play");
```

Comment:
You can just use `setAction("play")` here rather than `putExtra()` so you don't have to bother with both a key and a value. The action is basically just a key that doesn't need a value.